profiler/Directory.Build.props

```diff
@@ -188,9 +188,9 @@
 
 
     <PropertyGroup Condition=" '$(MSBuildProjectExtension)' == '.vcxproj' ">
-      <CORECLR-PATH>$(EnlistmentRoot)/profiler/src/dotnet-runtime-coreclr/</CORECLR-PATH>
       <DOTNET-TRACER-REPO-ROOT-PATH>$([System.IO.Path]::GetFullPath( $(EnlistmentRoot)/ ))</DOTNET-TRACER-REPO-ROOT-PATH>
-      <SHARED-LIB-PATH>$(DOTNET-TRACER-REPO-ROOT-PATH)shared/src/native-lib/</SHARED-LIB-PATH>
+      <SHARED-LIB-PATH>$(DOTNET-TRACER-REPO-ROOT-PATH)shared\src\native-lib\</SHARED-LIB-PATH>
+      <CORECLR-PATH>$(SHARED-LIB-PATH)coreclr\src</CORECLR-PATH>
       <SHARED-LIB-PLATFORM Condition="'$(Platform)'=='x64'">x64</SHARED-LIB-PLATFORM>
       <SHARED-LIB-PLATFORM Condition="'$(Platform)'=='Win32' OR '$(Platform)'=='x86'">x86</SHARED-LIB-PLATFORM>
       <SHARED-LIB-INCLUDES>$(SHARED-LIB-PATH)fmt_$(SHARED-LIB-PLATFORM)-windows-static\include;$(SHARED-LIB-PATH)spdlog\include</SHARED-LIB-INCLUDES>
```

Comment:
Total nitpick, but supposedly these could have been added at a higher level `Directory.Build.props` and inherited and reused in all `vcxproj` files (as we need them in all of them)? Not sure it's worth the time.

Reply:
Totally. If it's ok with you, we could do it in another PR (we will need 1 or 2 more) to continue the clean up.

Reply:
Sounds good! Resolving.